src/utils/getFilteredTodos.ts

```diff
@@ -0,0 +1,35 @@
+import { Todo } from '../types/Todo';
+import { CompletedFilter } from '../types/CompletedFilter';
+
+interface Props {
+  completedFilter: CompletedFilter;
+  searchQuary: string;
```

Comment:
There is a typo in variable name `searchQuary`. It should be `searchQuery`. It's always important to name variables correctly to avoid confusion in future.